deployment/conda/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+pip install docker-py fortranformat 'pint>=0.7' 'pyccc>=0.6.4' 'parmed>=2.7.3'
```

Comment:
Question: what does this part do?

Reply:
This installs packages that are not in anaconda.